Content.Client/Disposal/UI/DisposalUnitBoundUserInterface.cs

```diff
@@ -76,7 +73,9 @@ protected override void UpdateState(BoundUserInterfaceState state)
                 return;
             }
 
-            if (!IoCManager.Resolve<IEntityManager>().TryGetComponent(Owner.Owner, out DisposalUnitComponent? component)) return;
+            var entityManager = IoCManager.Resolve<IEntityManager>();
+            var entityId = Owner.Owner;
+            if (!entityManager.TryGetComponent(entityId, out DisposalUnitComponent? component)) return;
```

Comment:
same

Reply:
Didn't get this guy, as it's a BoundUserInterface, rather than an EntitySystem. Cleaned it up a bit, though.